integrations/wiz/.port/resources/port-app-config.yaml

```
@@ -0,0 +1,64 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: project
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"wizProject"'
+          identifier: .id
+          title: .name
+          properties:
+            slug: .slug
+            businessUnit: .businessUnit
+            businessImpact: .riskProfile.businessImpact
+  - kind: issue
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"wizIssue"'
+          identifier: .id
+          title: .entitySnapshot.name + " | " + .entitySnapshot.type
+          properties:
+            # https://integrate.wiz.io/reference/issues-query#:~:text=Direct%20a%20user%20to%20an%20Issue%20in%20Wiz
+            url: .id as $id | "https://app.wiz.io/issues#~(issue~'" + $id + ")"
```

dont set things to a var if you not doing anything complex with them

```suggestion
            url: "https://app.wiz.io/issues#~(issue~'" + .id + ")"
```

---

alright.

---

I attempted to follow your suggestion, but the port configuration failed to register due to its invalidity. Ultimately, I found that what successfully registered the port-app-config in Port was the `var` approach.

---

they syntax should be 
you had extra '
`"\"https://app.wiz.io/issues#~(issue~\" + .id + \")\""`

---

the  extra ' exists in the documentation

---

"\"https://app.wiz.io/issues#~(issue~\" + .id + \")\"" as well as other attempts I have made so far are failing to actually add the URL when i create the blueprint. This is why I kept the current way because it works alright. Lets stick with it.